pkg/suggestion/v1alpha3/hyperopt/base_hyperopt_service.py

```diff
@@ -9,9 +9,9 @@
 
 class BaseHyperoptService(object):
     def __init__(self, algorithm_name="tpe"):
-        if algorithm_name == 'hyperopt-tpe':
+        if algorithm_name == 'hyperopt-tpe' or algorithm_name == 'tpe':
```

Comment:
do we need "OR" condition? Since images do not have mapping, it is not possible to hit this condition at all.